### DISCUSSION TITLE
Making modifications to two/more collections within a single ACID transaction

### DISCUSSION THREAD

**Discussion by rameshkumar70** [pocketbase/pocketbase#3479] [Q&A] [answered]
Hi, I am using Sveltekit and I have three collections: Students, Sponsors, StudentSponsor (relationship collection to relate Students to Sponsors).  Suppose I wish to create a Student who has to have a Sponsor. I will therefore create a record in Student and another in StudentSponsor collections. As per my understanding of how databases work in general, I have to create the two records within a transaction, which should fail if any of the record creation fails. 
(At the same time, I also wish to leverage on the built-in authorization facilty that pocketbase provides to modify/read collections)

Reading Pocketbase's documentation, running transactions like this is not possible at the moment - correct me if I'm wrong. If yes, are there plans to provide this facility of running multiple collection operations within a single ACID transaction, in the future?
Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
Bulk/transaction operations are currently not supported with the web APIs (_this eventually will be implemented as part of #48 but it is left aside for now_).

At the moment you can execute transactions only if you [use PocketBase as framework](https://pocketbase.io/docs/use-as-framework/) and register your own [Go](https://pocketbase.io/docs/go-routing/#registering-new-routes) or [JS](https://pocketbase.io/docs/js-routing/#registering-new-routes) route. For a transaction example, you can also check:
- (Go) https://pocketbase.io/docs/go-records/#transaction
- (JS) https://pocketbase.io/docs/js-records/#transaction

**Reply by rameshkumar70** [reply]
I understand. Thank you for the prompt reply.
